src/servers/sapphire_zone/Zone/InstanceContent.cpp

```diff
@@ -31,6 +31,7 @@ Core::InstanceContent::InstanceContent( boost::shared_ptr< Core::Data::InstanceC
      m_instanceContentId( instanceContentId ),
      m_state( Created )
 {
+   m_instanceExpireTime = Util::getTimeSeconds() + ( m_instanceContentInfo->timeLimitmin * 60u );
```

Comment:
this is not correct, expire time should not be set until the duty actually starts.